scap/src/main.rs

```diff
@@ -62,22 +64,29 @@ fn main() {
         match frame {
             Frame::YUVFrame(frame) => {
                 println!(
-                    "Recieved frame {} of width {} and height {} and pts {}",
+                    "Recieved YUV frame {} of width {} and height {} and pts {}",
                     i, frame.width, frame.height, frame.display_time
                 );
             }
             Frame::BGR0(frame) => {
                 println!(
-                    "Received frame of width {} and height {}",
+                    "Received BGR0 frame of width {} and height {}",
                     frame.width, frame.height
                 );
+                println!("Frame data => {:?}", frame.data.len());
+                let x =
+                    image::RgbaImage::from_raw(frame.width as u32, frame.height as u32, frame.data);
+                let img = x.unwrap();
```

Comment:
Let's not add this here. We'll keep the main function easy to work with. 
We can add this under examples later